styles.css

```diff
@@ -694,3 +702,283 @@
   height: 34px;
   left: calc(50% - 375px / 2);
 }
+
+.deskto-line {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .navigation {
+    display: none;
+  }
```

Comment:
- [ ] You created very nice design, but there is one thing that need to be fixed. please reduce the weight of your container in order to match the desktop design. I pointed out in the below video👇
https://www.loom.com/share/bc0b35ef03b24fd2a9e667cf266e7077